cluster/gce/gci/configure.sh

```diff
@@ -248,7 +248,7 @@ function install-cni-binaries {
   fi
 
   echo "Downloading cni binaries"
-  download-or-bust "${cni_sha1}" "https://storage.googleapis.com/kubernetes-release/network-plugins/${cni_tar}"
+  download-or-bust "${cni_sha1}" "https://storage.googleapis.com/${CNI_STORAGE_PATH}/${cni_tar}"
```

Comment:
Seems like this isn't working, found error on: https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/79188/pull-kubernetes-e2e-gce/1141459405087182848/artifacts/e2e-c35fbe9ec1-674b9-master/kube-master-installation.log:
```
== Failed to download https://storage.googleapis.com/'kubernetes-release/network-plugins'/cni-plugins-amd64-v0.7.5.tgz. Retrying. ==
```

Reply:
Thanks, I have removed the single quotation mark '